src/app/donation-start/donation-start-form/donation-start-form.component.html

```diff
@@ -177,9 +176,8 @@ <h3 class="b-rt-0 b-m-0 b-bold"><span class="span-hr"></span>How does Big Give u
               </button>
             } @else {
               <button
-                style="width: 40%"
                 type="button"
-                class="continue b-w-100 b-rt-0"
```

Comment:
w-100 meant width 100%, was overridden by the inline style so deleted. I used scss `@extend` to do the `b-rt-0` via the continue class.